.github/config.yml

```diff
@@ -0,0 +1,29 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+# Heavily inspired by https://github.com/electron/electron/blob/master/.github/config.yml
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: |
+  👋 Thanks for opening your first issue here! If you're reporting a 🐞 bug, please make sure you include steps to reproduce it. We get a lot of issues on this repo, so please be patient and we will get back to you as soon as we can.
+  To help make it easier for us to investigate your issue, please follow the [contributing guidelines](https://github.com/videojs/video.js/blob/master/CONTRIBUTING.md) and [issue template](https://github.com/videojs/video.js/blob/master/.github/ISSUE_TEMPLATE.md).
+
+# Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
+
+# Comment to be posted to on PRs from first time contributors in your repository
+newPRWelcomeComment: |
+  💖 Thanks for opening this pull request! 💖
+  Things that will help get your PR across the finish line:
+
+  - Run `npm run lint -- --errors` locally to catch formatting errors earlier.
```

Comment:
maybe this should just mention that `npm run test` passes locally?

Reply:
it could also be worth mentioning that they should keep an eye on travis to make sure tests pass

Reply:
Maybe, I guess the question is how much we want to mention in this comment. We do already mention tests as well in the next bullet point, so, maybe that's enough reminder.